Purchases/Purchasing/StoreKitAbstractions/PromotionalOffer.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  PromotionalOffer.swift
+//
+//  Created by Josh Holtz on 1/18/22.
+
+import Foundation
+import StoreKit
+
+internal struct PromotionalOffer {
+    var identifier: String
+    var keyIdentifier: String
+    var nonce: UUID
+    var signature: String
+    var timestamp: Int
+
+    @available(iOS 12.2, macOS 10.14.4, watchOS 6.2, macCatalyst 13.0, tvOS 12.2, *)
+    var sk1SKPaymentDiscount: SKPaymentDiscount {
```

Comment:
Not: maybe move this to an extension to separate from the type?

Reply:
nit: can we just name this var `sk1PromotionalOffer` and/or create a typealias, so we don't have to worry about the underlying type name differences between sk1 and sk2?

Reply:
I agree, I'd name this `sk1PromotionalOffer`.